blobs.go

```diff
@@ -142,17 +144,28 @@ func (i *blobsByHashIter) Next() (sql.Row, error) {
 			return nil, err
 		}
 
-		return blobToRow(blob), nil
+		return blobToRow(blob)
 	}
 }
 
 func (i *blobsByHashIter) Close() error {
 	return nil
 }
 
-func blobToRow(c *object.Blob) sql.Row {
+func blobToRow(c *object.Blob) (sql.Row, error) {
+	r, err := c.Reader()
+	if err != nil {
+		return nil, err
+	}
+
+	content, err := ioutil.ReadAll(r)
```

Comment:
Is it ok to load all the content in memory for all the blobs?

Reply:
Only one file at a time is ok I think.

Reply:
If the connection sends the rows in streaming, it should only send 1 at a time, which is ok. If not, it can be huge, yes.

How should we specify the threshold? It should be configurable.

Why limit to non-binary files only? Maybe you want to get a binary file like an image or something. Even if we're gonna use it for code analysis, this is general purpose and someone might want to idk, query for images and get their contents.

Reply:
we can leave for the first version two constants, one to only get binary files, and another with a threshold to do not send files bigger than X MB.

Reply:
Should we at least have some environment variables for now with a default?

Reply:
Ok, but just for now.

Reply:
Open an issue for configuration: https://github.com/src-d/go-mysql-server/issues/109